src/App.css

```diff
@@ -1,18 +1,124 @@
+:root {
+  --primary-color: #1a73e8;
+  --hover-primary: #1557b0;
+  --background-color: #f8f9fa;
+  --column-background: #ffffff;
+  --border-color: #dadce0;
+  --text-primary: #202124;
+  --text-secondary: #5f6368;
+}
+
 .app {
-  padding: 20px;
+  padding: 24px;
+  background-color: var(--background-color);
+  min-height: 100vh;
+}
+
+.app h1 {
+  color: var(--text-primary);
+  font-size: 24px;
+  font-weight: 400;
+  margin-bottom: 24px;
+}
+
+.forms-container {
+  display: flex;
+  gap: 16px;
+  margin-bottom: 32px;
+  flex-wrap: wrap;
+}
+
+.task-form,
+.column-form {
+  display: flex;
+  gap: 8px;
+}
+
+.task-form input,
+.column-form input {
+  padding: 8px 16px;
+  border: 1px solid var(--border-color);
+  border-radius: 24px;
+  font-size: 14px;
+  outline: none;
+  min-width: 200px;
+  transition: all 0.2s ease;
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
transition: border-color 0.2s ease, box-shadow 0.2s ease, background-color 0.2s ease;
```
The transition property on inputs and buttons should be more specific about which properties to animate for better performance. Instead of using 'all', specify the exact properties that need transitions.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;